src/app/note/create-edit-note/create-edit-note.component.ts

```diff
@@ -43,7 +44,8 @@ export class CreateEditNoteComponent implements OnInit, OnDestroy, AfterViewInit
   projectIds: Array<any> = [];
   codeSampleDialogHeight = window.innerHeight * 0.8
 
-  constructor(@Host() parent: ModalService,
+  constructor(
+    private parent: ModalService,
```

Comment:
Why we are injecting 2 instance of ModalService here? What's parent supposed to do here? I see it was an instance coming from Host before. let's discuss about it.